sdk/cosmosdb/Azure.ResourceManager.CosmosDB/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2023-02-16)
```

Comment:
Please just leave the release date as `Unreleased`.
We could change it when we are about to make the release